packages/next/index.d.ts

```diff
@@ -0,0 +1,14 @@
+/// <reference types="./types/global" />
+/// <reference path="./amp.d.ts" />
+/// <reference path="./app.d.ts" />
+/// <reference path="./config.d.ts" />
+/// <reference path="./document.d.ts" />
+/// <reference path="./dynamic.d.ts" />
+/// <reference path="./error.d.ts" />
+/// <reference path="./head.d.ts" />
+/// <reference path="./image.d.ts" />
+/// <reference path="./link.d.ts" />
+/// <reference path="./router.d.ts" />
+/// <reference path="./script.d.ts" />
+
+export * from './types'
```

Comment:
Usually you would never see this fail while developing if the editor navigates directly to the definition (like VS Code), so it should feel very natural, like nothing changed but now you have access to more modules.